### ISSUE TITLE
Loop Out Taking Forever

### ISSUE THREAD

**Issue by emretezel** [lightninglabs/loop#386]
Hi Everyone,

Today at 12:42p London time I initiated a Loop Out on a channel with Moon using LiT, the transaction ID is

```
11FB9F98723B7DE908538A54B188B7F04AA75C1F22E4CACC27E162085654AC95
```

The operation is stuck in Initiated phase, the channel on Terminal has green colour but it has 0 "Can Receive" SATs.

Is this is expected, is there timeout when the operation will fail and I will have the SATs returned locally to the channel. 

```
lncli trackpayment 11FB9F98723B7DE908538A54B188B7F04AA75C1F22E4CACC27E162085654AC95
+-------------------------------+--------------+--------------+--------------+-----------+----------+--------------------+------------------------------------------------------------------------------------------------------+
| HTLC_STATE                    | ATTEMPT_TIME | RESOLVE_TIME | RECEIVER_AMT | FEE       | TIMELOCK | CHAN_OUT           | ROUTE                                                                                                |
+-------------------------------+--------------+--------------+--------------+-----------+----------+--------------------+------------------------------------------------------------------------------------------------------+
| TEMPORARY_CHANNEL_FAILURE @ 1 |        7.463 |        7.912 | 1081347      | 9681.146  |   683250 | 750878480876044288 | Moon (paywithmoon.com)->Solmyr->LOOP->03fb2a                                                         |
| TEMPORARY_CHANNEL_FAILURE @ 3 |        9.657 |       15.632 | 1081347      | 6.243     |   683330 | 750878480876044288 | Moon (paywithmoon.com)->02fb22354383c240f9a0->1ML.com node ALPHA->032ff5bfb4b9428ef33d->LOOP->03fb2a |
| TEMPORARY_CHANNEL_FAILURE @ 1 |       17.040 |       17.501 | 1081347      | 329.649   |   683412 | 750878480876044288 | Moon (paywithmoon.com)->LNBIG.com [lnd-07]->LNBIG.com [lnd-05]->032ff5bfb4b9428ef33d->LOOP->03fb2a   |
| TEMPORARY_CHANNEL_FAILURE @ 1 |       19.242 |       19.686 | 1081347      | 7577.586  |   683325 | 750878480876044288 | Moon (paywithmoon.com)->Rahula Node->smallworlnd->LOOP->03fb2a                                       |
| TEMPORARY_CHANNEL_FAILURE @ 1 |       21.401 |       21.845 | 1081347      | 6491.169  |   683354 | 750878480876044288 | Moon (paywithmoon.com)->ACINQ->LOOP->03fb2a                                                          |
| TEMPORARY_CHANNEL_FAILURE @ 3 |       23.493 |       27.968 | 1081347      | 329.649   |   683412 | 750878480876044288 | Moon (paywithmoon.com)->LNBIG.com [lnd-17]->LNBIG.com [lnd-05]->032ff5bfb4b9428ef33d->LOOP->03fb2a   |
| TEMPORARY_CHANNEL_FAILURE @ 1 |       29.736 |       30.188 | 1081347      | 8112.291  |   683260 | 750878480876044288 | Moon (paywithmoon.com)->SilentBob->LOOP->03fb2a                                                      |
| IN_FLIGHT                     |       31.859 |            - | 1081347      | 10816.562 |   683250 | 750878480876044288 | Moon (paywithmoon.com)->yalls.org clearnet-yalls->LOOP->03fb2a                                       |
+-------------------------------+--------------+--------------+--------------+-----------+----------+--------------------+------------------------------------------------------------------------------------------------------+
Amount + fee:   0 + 0 sat
Payment hash:   11fb9f98723b7de908538a54b188b7f04aa75c1f22e4cacc27e162085654ac95
Payment status: IN_FLIGHT
```

Best,

**Comment by alexbosworth** [comment]
Thanks for the report

Can you confirm that you have the process running? You shouldn't close the loop process during the swap

**Comment by emretezel** [comment]
I believe loop runs integrated to Lightning Terminal and the Terminal is running.

**Comment by emretezel** [comment]
The swap has terminated with the following error message

```
Sweep Timeout
The on-chain HTLC wasn't swept before the server revoked the HTLC.
```

**Comment by emretezel** [comment]
I don't know how can I get the on-chain address for the swap, which command.

I have increased loop fee settings hopefully will have more luck
```
# Loop settings
loop.maxlsatcost=10000
loop.maxlsatfee=1000
loop.loopoutmaxparts=10
```

**Comment by carlaKC** [comment]
Thanks for the issue report @emretezel!

I looked up your swap on our side, and it appears that the server broadcast the swap htlc but your loop client did not sweep it. Once the on-chain htlc expired, we swept the timeout clause and canceled your swap payment invoice. 

The HTLC confirmed with a lot of time for your sweep tx to come through, so I'm curious as to why the client didn't sweep. Do you have any logs that say "Required fee x exceeds max miner fee of y" for this swap? That would explain why your client chose not to sweep the success path and complete the swap. 

As for your swap payment, we've recently patched a bug in lnd where payments can get stuck in-flight when they fail. Funds have been returned to your node, but the payment itself just needs to have its state updated. This will be fixed with the upcoming 0.13 lnd release, so for now you may have to live with the swap being stuck, and it'll resolve once you upgrade lnd.

**Comment by carlaKC** [comment]
Closing due to inactivity. 

Feel free to re-open with extra information from loop's logs.
